lib/absinthe/schema/notation.ex

```diff
@@ -992,9 +992,11 @@ defmodule Absinthe.Schema.Notation do
   """
   defmacro import_types(type_module_ast) do
     env = __CALLER__
-    type_module_ast
-    |> Macro.expand(env)
-    |> do_import_types(env)
+    {:ok, _} =
+      type_module_ast
+      |> Macro.expand(env)
+      |> do_import_types(env)
```

Comment:
I guess, it was returning `{:ok, tuple}`  where the tuple had some structs not actually quoted expressions, and  when the macro result was being expanded an error: `%Absinthe.Schema.Notation.Definition{...} is not a valid quoted expression` was raised. Looks like previous elixir did not failed with this kind of things :/